### ISSUE TITLE
PSVITA is not connected!

### ISSUE THREAD

**Issue by dnarock666** [lazydevyo/PSVitaPlay#16]
- I use enso 3.60
- I've got an OLED non-3g model
- I downloaded the udcd_uvc.skprx file into the ur0:tai/ folder
- I added the correct line in the config.txt below the kernel rows (ur0:tai/udcd_uvc.skprx)
- I don't have a tai folder in the ux0
- I plugged the jack cable to the front mic input
- I set the "Microphone Line-In Front" as default recording input
- I renamed the "Microphone Line-In Front PSVita"

I ran the PSVitaDock.exe executable inside the PSVitaPlay.v4.x86_64 unzipped folder and it shows to me the "PSVITA is not connected!" message, and a audio icon in the right-bottom corner keep coming.

What's happening?

![image](https://user-images.githubusercontent.com/15653624/68628537-2ac61a00-04e1-11ea-9d58-e165af0ed990.png)

**Comment by GhsVilela** [comment]
I have the same problem, did you found a solution?

**Comment by dnarock666** [comment]
> I have the same problem, did you found a solution?

Yes, but it's the ugliest possible. I found that my PS Vita has not got a product number under her shell and it's not compatible with Data Sync.
You can try anything possible by software but the tragic solution if you don't have a EUR/USA/JAP Product Number, is to buy another PS VITA.
Like I did :'(

**Comment by GhsVilela** [comment]
Thanks for the reply, I guess I'll just give up haha
But the problem that I'm having right now it's that the PS Vita reboots when the plugin is installed

**Comment by AbyssHun** [comment]
> Thanks for the reply, I guess I'll just give up haha
> But the problem that I'm having right now it's that the PS Vita reboots when the plugin is installed

sorry to say but you may not have had to get a new vita. i was having the same issue and moved the plugin to the uro:/ tai folder and then put the plugin path as the last kernel and was able to boot it fine.

**Comment by sudo-apt-get-install-ethan** [comment]
> > Thanks for the reply, I guess I'll just give up haha
> > But the problem that I'm having right now it's that the PS Vita reboots when the plugin is installed
> 
> sorry to say but you may not have had to get a new vita. i was having the same issue and moved the plugin to the uro:/ tai folder and then put the plugin path as the last kernel and was able to boot it fine.

didnt work for me.. i did the exact same thing. help?
